Added/Improved Intros for the Map Topics (#6887)
github/docs: content/github/getting-started-with-github/using-github/index.md

---
title: Using GitHub

> intro: 'Explore {% data variables.product.company_short %}'s products from different platforms and devices.'

Reviewer: ```suggestion
intro: "Explore {% data variables.product.company_short %}'s products from different platforms and devices."
```

> intro: '{% data variables.product.prodname_dotcom %} is accessible in multiple platforms and devices where you can explore the features and products of it.'

Reviewer: ```suggestion
intro: 'Explore {% data variables.product.company_short %}'s products from different platforms and devices.'
```
